lighthouse-core/audits/installable-manifest.js

```diff
@@ -61,8 +61,8 @@ const UIStrings = {
   */
   'no-acceptable-icon': `No supplied icon is at least {value0}\xa0px square in PNG, SVG or WebP format`,
 
-  /** Error message explaining that the downloaded icon was empty or corrupt. */
-  'cannot-download-icon': `Downloaded icon was empty or corrupted`,
+  /** Error message explaining that an icon was unable to be downloaded. */
+  'cannot-download-icon': `Icon was unable to be downloaded`,
```

Comment:
@adrianaixba @paulirish I'm assuming this is a better string based on the `cannot-download-icon ` `errorId`